megablocks/layers/glu.py

```diff
@@ -1,5 +1,6 @@
 from megablocks.layers import common
 from megablocks.layers import gelu
+from megablocks.layers.activation_fn import act_fn
```

Comment:
Did you mean to add this module? I don't see it, and I think you don't need it because the argument is set to a callable?

Reply:
Forgot to add

It's effectively what you do for [gelu](https://github.com/stanford-futuredata/megablocks/blob/main/megablocks/layers/gelu.py#L33) but takes in a generic callable function